pkg/api/folder.go

```diff
@@ -175,7 +177,7 @@ func (hs *HTTPServer) DeleteFolder(c *models.ReqContext) response.Response { //
 	}
 
 	uid := web.Params(c.Req)[":uid"]
-	f, err := hs.folderService.DeleteFolder(c.Req.Context(), c.SignedInUser, c.OrgID, uid, c.QueryBool("forceDeleteRules"))
+	f, err := hs.folderService.DeleteFolder(c.Req.Context(), &folder.DeleteFolderCommand{UID: uid})
```

Comment:
add orgID

Reply:
hm, `folder.DeleteFolderCommand` doesn't have an OrgID field

Reply:
that should be added on the struct